pkg/csource/options.go

```diff
@@ -210,31 +211,31 @@ func DeserializeOptions(data []byte) (Options, error) {
 	data = bytes.Replace(data, []byte("Sandbox: "), []byte("Sandbox:empty "), -1)
 	waitRepeat, debug := false, false
 	n, err := fmt.Sscanf(string(data),
-		"{Threaded:%t Collide:%t Repeat:%t Procs:%d Sandbox:%s"+
+		"{Threaded:%t Collide:%t Repeat:%t Procs:%d Sandbox:%s SandboxArg:%d"+
 			" Fault:%t FaultCall:%d FaultNth:%d EnableTun:%t UseTmpDir:%t"+
 			" HandleSegv:%t WaitRepeat:%t Debug:%t Repro:%t}",
-		&opts.Threaded, &opts.Collide, &opts.Repeat, &opts.Procs, &opts.Sandbox,
+		&opts.Threaded, &opts.Collide, &opts.Repeat, &opts.Procs, &opts.Sandbox, &opts.SandboxArg,
 		&opts.Fault, &opts.FaultCall, &opts.FaultNth, &opts.NetInjection, &opts.UseTmpDir,
 		&opts.HandleSegv, &waitRepeat, &debug, &opts.Repro)
 	if err == nil {
-		if want := 14; n != want {
-			return opts, fmt.Errorf("failed to parse repro options: got %v fields, want %v", n, want)
+		if want := 15; n != want {
```

Comment:
Note that this function tries to parse all legacy formats. Now we have +1 legacy format, so it should now consider 3 alternatives instead of two. Or it should be somehow refactored so that it supports all of them.

The corresponding test should verify that the parser is still able to parse all previous formats and the current one (with sandbox arg).